modules/vpc-baseline/variables.tf

```diff
@@ -8,16 +8,25 @@ variable "enable_flow_logs" {
   default     = true
 }
 
+variable "vpc_log_destination_type" {
+  description = "The type of the logging destination. Valid values: cloud-watch-logs, s3"
+  default     = "cloud-watch-logs"
+}
+
+variable "vpc_flow_logs_s3_arn" {
```

Comment:
Could you add a default value here as well?

Reply:
@nozaq addressed! I'm not 100% what the default value should be since it's an ARN to an externally created resource, but I've put a sample one, let me know if this is what you had in mind

Reply:
Appreciate a quick response!
I guess this could be an empty string just as set [here](https://github.com/nozaq/terraform-aws-secure-baseline/blob/8f9e11ad688e1e32f145f56666cdca428212e6b2/variables.tf#L224)?
The reason to have a default value is just to make this variable optional, otherwise it would be treated as required.

Reply:
Ah indeed, makes sense. Addressed